internal/colorize/wrap_test.go

```diff
@@ -7,82 +7,82 @@ import (
 	"testing"
 )
 
-func Test_GetCroppedText(t *testing.T) {
```

Comment:
Please add tests covering the new mechanic. Particularly color codes not covering the "continuation".